vllm/model_executor/models/phi3v.py

```diff
@@ -321,6 +321,17 @@ def get_phi3v_image_feature_size(
         + (new_height // 336 + 1) * 12
 
 
+def get_max_phi3v_image_tokens(ctx: InputContext):
```

Comment:
`get_max_phi3v_image_tokens` and `dummy_data_for_phi3v` are both based on `dummy_height, dummy_width = 8000, 50`, so we should make these constants to this file for consistency. I think this will suffice for the purpose of consistency for now, and in the future we can establish more structured protocol between multimodal feature size and dummy data.

Reply:
I've amde #6146 to address this.